### PR TITLE
update versions and changelog for 0.6.0

### PR DESCRIPTION
**Pull request by conradoplg**

### CODE REVIEW

File: frost-core/CHANGELOG.md [hunk 1]
```diff
@@ -23,8 +27,14 @@ Entries are listed in reverse chronological order.
 * `SignatureResponse` was removed. `SignatureShare` can now be encoded directly with
   `from/to_bytes()`.
 * rename all `to_bytes()`/`from_bytes()` to `serialize()`/`deserialize()`
-
-## Released
+* The group public key is now included in the hash inside the binding factor
+  computation. This reflects an upcoming change to the specification:
+  https://github.com/cfrg/draft-irtf-cfrg-frost/pull/439
+* `generate_with_dealer()` was change to allow specifying which identifiers to use
+* Identifiers can now be derive from arbitrary strings with `Identifier::derive()`
```
Comment (natalieesk) [inline frost-core/CHANGELOG.md:34]: typo

File: frost-core/CHANGELOG.md [hunk 2]
```diff
@@ -4,6 +4,10 @@ Entries are listed in reverse chronological order.
 
 ## Unreleased
 
+## 0 7.0
```
Comment (natalieesk) [inline frost-core/CHANGELOG.md:7]: typo

**Review by natalieesk — CHANGES_REQUESTED** [review]
Just a couple of typos

File: frost-core/CHANGELOG.md (hunk 2 above)
Comment (conradoplg) [reply frost-core/CHANGELOG.md]: fixed in 5071396

**Review by natalieesk — APPROVED** [review]

**Review by natalieesk — APPROVED** [review]

**Pull request merged**